formats/README.md

```diff
@@ -59,13 +59,12 @@ This library allows serialization and deserialization of objects to and from [Av
 
 Allows serialization and deserialization of objects to and from [BSON](https://docs.mongodb.com/manual/reference/bson-types/).
 
-### Ktoml 
+### TOML (Multiplatform) 
 * GitHub repo: [akuleshov7/ktoml](https://github.com/akuleshov7/ktoml)
 * Artifact ID: `com.akuleshov7:ktoml-core`
-* Platforms: multiplatform, all Kotlin supported platforms
+* Platforms: all supported platforms
 
-Fully Native and Multiplatform Kotlin serialization library for serialization/deserialization of TOML format.
-This library contains no Java code and no Java dependencies and it implements multiplatform parser, decoder and encoder of TOML.
```

Comment:
Isn't this line also relevant?

Reply:
Yes, just want to make the title be like others.

Reply:
Not all format descriptions fit in one line or sentence (see Avro or XML), so IMO it's better to leave it as is